tfjs-core/src/ops/draw_test.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC. All Rights Reserved.
```

Comment:
no all rights reserved.
you can use https://github.com/chunnienc/tfjs-license-fix to add license headers.

Reply:
Done.